src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -129,6 +129,7 @@ export default class Preview extends React.Component<Props> {
             return;
         }
 
+        this.updatePreview.clear();
```

Comment:
What does that do? :thinking:

Reply:
this stops/clears the `Debounce` object that is used to update the preview. this prevents an error if the preview is unmounted while there is still a "pending" update-request. 

this case happens for example if a change in the form fires an update of the preview and hides the preview because of the `previewCondition`.